src/Common/src/Interop/Windows/kernel32/Interop.GetCPInfoEx.cs

```diff
@@ -8,7 +8,7 @@ internal partial class Interop
 {
     internal partial class Kernel32
     {
-        private const uint CP_ACP = 0x0u;
```

Comment:
`CP_ACP` is the Windows SDK name for this. It should stay that way.

The convention is to use the same name as what's used in the native code (https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/interop-guidelines.md).